app/src/main/java/projekt/substratum/fragments/ProfileFragment.java

```diff
@@ -257,6 +257,7 @@ public View onCreateView(
                         backupFunction.execute();
                         Log.d(References.SUBSTRATUM_LOG, selectedBackup.toString());
                         dialog.dismiss();
+                        backup_name.setText("");
```

Comment:
Read up a bit, apparently it's the recommended way to clear `EditView`s on Android. Relevant commit pushed.